custom_components/opnsense/switch.py

```
@@ -10,241 +11,292 @@
+        _compile_static_switches(config_entry, coordinator, state),
+        return_exceptions=True,
     )
-    cem.process_entities()
+
+    entities: list = []
+    for result in results:
+        if isinstance(result, list):
+            entities += result
+        else:
+            _LOGGER.error(
+                f"Error in switch async_setup_entry. {result.__class__.__qualname__}: {result}"
```

Ruff

Logging statement uses f-string

(G004)